src/styles/globalStyles.js

```diff
@@ -43,7 +43,7 @@ export default createGlobalStyle`
           color: ${theme.colors.grayDark};
         `};
     }
-    h3,p{
+    h3,p,li{
```

Comment:
here as well, p and li should have the default text styling. I don't think you need to add them to the global styles explicitly